### PR TITLE
feat: add Product detail page to E-commerce

### PR DESCRIPTION
**Pull request by mariyasada**
add Product detail page to E-commerce
adding this page's  link to the product list page
importing CSS files to main files

### CODE REVIEW

File: stylesheets/product-detail/textutil.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.text-size-md
+{
+    font-size: 2rem;
+}
+.text-red
+{
+    color:red;
```
Comment (SuyashTiwariCoder) [inline stylesheets/product-detail/textutil.css]: You can use variables , for colors .

File: stylesheets/product-detail/textutil.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+}
+.text-black
+{
+    color: black;
```
Comment (SuyashTiwariCoder) [inline stylesheets/product-detail/textutil.css]: same here , You can store your color in a variable then use it evrywhere .

**Review by SuyashTiwariCoder — APPROVED** [review]
Other than thay , everything looks Good to me .

**Review by munjalSaksham71 — APPROVED** [review]
Looks good, just need to make changes on what suyash suggested. Rest looks good.

File: stylesheets/product-detail/textutil.css (hunk 1 above)
Comment (mariyasada) [reply stylesheets/product-detail/textutil.css]: done

**Pull request merged**